src/pages/User/components/PullRequests/ShareButtons.js

```diff
@@ -14,7 +14,7 @@ const ShareButtons = ({ username, pullRequestCount }) => (
       <a
         target="_blank"
         rel="noopener noreferrer"
-        className="bg-mid-blue hover:bg-light-pink transition text-white hover:text-white px-2 py-1 pointer no-underline text-sm"
+        className="bg-mid-blue hover:bg-light-pink transition-btn text-white hover:text-white px-2 py-1 pointer no-underline text-sm"
```

Comment:
Can we use https://tailwindcss.com/docs/transition-duration rather than our own class? So `transition duration-300` ?

Reply:
noted